src/main/scala/mesosphere/marathon/core/task/bus/TaskBusModule.scala

```diff
@@ -0,0 +1,16 @@
+package mesosphere.marathon.core.task.bus
+
+import mesosphere.marathon.core.task.bus.impl.{
+  TaskStatusEmitterImpl,
+  TaskStatusObservablesImpl,
+  InternalTaskStatusEventStream
+}
+
+class TaskBusModule {
```

Comment:
Please add some documentation for this module as well the provided objects/functionality.